crates/service_utils/src/helpers.rs

```diff
@@ -401,3 +411,91 @@ pub fn parse_config_tags(
         }
     }
 }
+
+pub async fn execute_webhook_call<T>(
+    webhook_object: Value,
+    payload: T,
+    config_version_opt: &Option<String>,
+    event: WebhookEvent,
+) -> Result<(), AppError>
+where
+    T: Serialize,
+{
+    match serde_json::from_value::<Webhook>(webhook_object) {
+        Ok(webhook) => {
+            let http_client = reqwest::Client::new();
```

Comment:
We can add this to AppState, creating a new HttpClient is expensive operation. This can then be used everywhere else in Superposition